tests/codegen.rs

```diff
@@ -35,6 +35,7 @@ fn run_test_for_path(path: &str) {
 #[derive(Debug)]
 enum Test {
     Check(String),
+    CheckAbsent(String),
     NotCheck(String),
```

Comment:
When should I use `CHECK-ABSENT` and when should I use `NOT-CHECK`?

Reply:
I've added some comments